recipes/folly/all/conan_deps.cmake

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.17)
+# Set the dependency flags expected by https://github.com/facebook/folly/blob/v2023.12.18.00/CMake/folly-deps.cmake
+
+macro(custom_find_package package_name variable_prefix)
+    find_package(${package_name} REQUIRED CONFIG ${ARGN}
+        # Allow only Conan packages
+        NO_DEFAULT_PATH
+        PATHS ${CMAKE_PREFIX_PATH}
```

Comment:
```suggestion
```

unless there's actual issues, I would remove this.

This would break the ability to use https://docs.conan.io/2/reference/config_files/profiles.html#platform-requires -